styleguide/derek/examples/subpanels.ejs

```diff
@@ -0,0 +1,158 @@
+  'business-resources': {'title': 'Business Resources'},
+  'cta': {'title': 'CTA'},
+  'features': {'title': 'Features'},
+  'services-3-col': {'title': 'IAS: 3 cols'},
+  'services-4-col': {'title': 'IAS: 4 cols'},
+  'services-5-col': {'title': 'IAS: 5 cols'},
+  'services-6-col': {'title': 'IAS: 6 cols'},
+  'key-offers': {'title': 'Key Offers'},
+  'lead': {'title': 'Lead'},
+  'list': {'title': 'List'},
+  'testimonials': {'title': 'Quote'},
+  'text': {'title': 'Text'},
+  'value-prop': {'title': 'Value Prop'}
+};
+%>
+
+<div class="container-fluid">
+  <div class="row standard-padding-full center-block">
+    <div class="col-md-12">
+      <h5 class="red">Global Ecomm Subpanels</h5>
+      <h2 class="no-margin">Yo dawg, I heard you like panels, so we put panels inside of panels.</h2>
+    </div>
+  </div>
+</div>
+
+<div id="pattern-selector" class="bg-steel-blue container-fluid half-padding-full">
+  <div class="row center-block">
+    <div class="col-sm-2">
+      <select id="mp-layout" name="layout">
+        <% for (var layout in layouts) { %>
+        <option value="<%- layout %>"><%- layouts[layout].title %></option>
+        <% } %>
+      </select>
+    </div>
+    <div class="col-sm-3">
+      <select id="mp-pattern-left" name="pattern_left">
+        <% for (var pattern in patterns) { %>
+        <option value="<%- pattern %>">[Left] <%- patterns[pattern].title %></option>
+        <% } %>
+      </select>
+    </div>
+    <div class="col-sm-3">
+      <select id="mp-pattern-right" name="pattern_right">
+        <% for (var pattern in patterns) { %>
+        <option value="<%- pattern %>">[Right] <%- patterns[pattern].title %></option>
+        <% } %>
+      </select>
+    </div>
+    <div class="col-sm-4">
+      <input type="submit" class="btn btn-xs btn-default" id="search-mini-panels" value="Go to Panel"/>
+      <input type="reset" class="btn btn-xs btn-info" id="reset-mini-panels" value="Back to Top"/>
+    </div>
+  </div>
+</div>
+
+<%
+for (var layout in layouts) {
+  for (var pattern_left in patterns) {
+    for (var pattern_right in patterns) {
+      var anchor_id = 'pattern_' + layout + '_' + pattern_left + '_' + pattern_right;
+      var class_left = layouts[layout].class_left;
+      var class_right = layouts[layout].class_right;
+      var section_title = layouts[layout].title + ' (' + patterns[pattern_left].title + '/' + patterns[pattern_right].title + ')';
+      var file_left = '../_partials/solutions/' + pattern_left;
+      var file_right = '../_partials/solutions/' + pattern_right;
+%>
+
+<div id="<%- anchor_id %>" class="container half-padding">
+  <h4><%- section_title %></h4>
+  <div class="row">
+    <div class="<%- class_left %>">
+      <%- partial(file_left, {isSubpanel: true}) %>
+    </div>
+    <div class="<%- class_right %>">
+      <%- partial(file_right, {isSubpanel: true}) %>
+    </div>
+  </div>
+</div>
+<hr/>
+<%
+    }
+  }
+}
+%>
```

Comment:
The above essentially creates every combination of layout, left pattern and right pattern and includes the necessary files, all while setting `isSubpanel` to `true`, so we can see our pattern alternatives.